homeassistant/components/govee_ble/sensor.py

```diff
@@ -116,13 +117,21 @@ async def async_setup_entry(
 
 class GoveeBluetoothSensorEntity(
     PassiveBluetoothProcessorEntity[
-        PassiveBluetoothDataProcessor[Optional[Union[float, int]]]
+        PassiveBluetoothDataProcessor[Optional[Union[float, int, str]]]
```

Comment:
Without this we get
`homeassistant/components/govee_ble/sensor.py:130: error: Non-overlapping equality check (left operand type: "Union[float, int, None]", right operand type: "str")  [comparison-overlap]`

Or do you prefer
`self.processor.entity_data.get(self.entity_key) != ERROR  # type: ignore[comparison-overlap]` ?

Reply:
better to do it as you have it and avoid the ignore